mozci/util/evaluate_batch_execution.py

```diff
@@ -0,0 +1,242 @@
+# -*- coding: utf-8 -*-
+import argparse
+import csv
+import json
+import os
+
+from loguru import logger
+
+from mozci.push import MAX_DEPTH, Push, PushStatus
+from mozci.util.classify_batch_execution import BASE_OUTPUT_DIR, PARAMETERS_NAMES
+
+CSV_HEADER = [
+    "configuration",
+    "total_pushes",
+    "classify_errors",
+    "evaluate_errors",
+    "correct_good",
+    "wrong_good",
+    "correct_bad",
+    "wrong_bad",
+    "total_real",
+    "correct_real",
+    "wrong_real",
+    "missed_real",
+    "total_intermittent",
+    "correct_intermittent",
+    "wrong_intermittent",
+    "missed_intermittent",
+]
+
+
+def evaluate_groups(
+    group_summaries, predicted_groups, sheriff_groups, expected, suffix
+):
+    """
+    Compare annotated groups with predicted ones to output an evaluation with total
+    (predicted)/correctly (predicted)/wrongly (predicted)/missed (annotated) counts
+    """
+    total = len(predicted_groups)
+    if not total:
+        return {
+            f"total{suffix}": 0,
+            f"correct{suffix}": 0,
+            f"wrong{suffix}": 0,
+            f"missed{suffix}": len(sheriff_groups),
+        }
+
+    differing = 0
+    for group in predicted_groups:
+        classifications_set = set(
+            task.classification for task in group_summaries[group].tasks if task.failed
+        )
+        if len(classifications_set) == 0:
+            continue
+
```

Comment:
We should exclude pending and conflicting from the evaluation, to avoid skewing the results.
I wonder if we can somehow share code with the eval command to avoid duplication.